tests/foreman/cli/test_contentview.py

```diff
@@ -188,6 +188,28 @@ def test_negative_create_with_org_name(self):
         with self.assertRaises(CLIReturnCodeError):
             ContentView.create({'organization-id': gen_string('alpha')})
 
+    @tier1
+    def test_positive_create_empty_and_verify_files(self):
```

Comment:
The cv would not be implicit by being on this module?

Same as test_postive_create_with_name